frontend/app/component/form/differences/DifferencesForm.test.tsx

```diff
@@ -240,13 +217,127 @@ describe("Test DifferencesForm", () => {
+          warnings: [],
+        },
+      });
+
       const user = userEvent.setup();
 
       render(Component);
 
-      await user.type(screen.getByTestId("more_ballots_count"), "2");
+      // Since the component does not allow to change values in other components,
+      // not inputting any values and just clicking the submit button.
+      const submitButton = screen.getByRole("button", { name: "Volgende" });
+      await user.click(submitButton);
+
+      const feedbackError = await screen.findByTestId("feedback-error");
+      expect(feedbackError).toHaveTextContent(/^IncorrectDifference$/);
+      expect(screen.queryByTestId("feedback-warning")).toBeNull();
+      expect(screen.queryByTestId("server-feedback-error")).toBeNull();
+    });
+
+    test("F.23/24 IncorrectDifference", async () => {
+      overrideOnce("post", "/api/polling_stations/1/data_entries/1", 200, {
+        validation_results: {
+          errors: [
+            {
+              fields: ["data.differences_counts.fewer_ballots_count"],
+              code: "IncorrectDifference",
+            },
+          ],
+          warnings: [],
+        },
+      });
+
+      const user = userEvent.setup();
+
+      render(Component);
+
+      // Since the component does not allow to change values in other components,
+      // not inputting any values and just clicking the submit button.
+      const submitButton = screen.getByRole("button", { name: "Volgende" });
+      await user.click(submitButton);
+
+      const feedbackError = await screen.findByTestId("feedback-error");
+      expect(feedbackError).toHaveTextContent(/^IncorrectDifference$/);
+      expect(screen.queryByTestId("feedback-warning")).toBeNull();
+      expect(screen.queryByTestId("server-feedback-error")).toBeNull();
+    });
+
+    test("F.25 ConflictingDifferences", async () => {
+      overrideOnce("post", "/api/polling_stations/1/data_entries/1", 200, {
+        validation_results: {
+          errors: [
+            {
+              fields: [
+                "data.differences_counts.more_ballots_count",
+                "data.differences_counts.fewer_ballots_count",
+              ],
+              code: "ConflictingDifferences",
+            },
+          ],
+          warnings: [],
+        },
+      });
+
+      const user = userEvent.setup();
+
+      render(Component);
+
+      // Since the component does not allow to change values in other components,
+      // not inputting any values and just clicking the submit button.
+      const submitButton = screen.getByRole("button", { name: "Volgende" });
+      await user.click(submitButton);
+
+      const feedbackError = await screen.findByTestId("feedback-error");
+      expect(feedbackError).toHaveTextContent(/^ConflictingDifferences$/);
+      expect(screen.queryByTestId("feedback-warning")).toBeNull();
+      expect(screen.queryByTestId("server-feedback-error")).toBeNull();
+    });
+
+    test("F.26 Incorrect total", async () => {
```

Comment:
Tests should use `overrideOnce()` and not rely on the mock implementation.

Reply:
Changed in b90d2a8d33a757b4a0ada6d31a3ade9b59aab4f8